docker/Dockerfile

```diff
@@ -1,12 +1,11 @@
 ARG BRANCH=${BRANCH:-ce}
 ARG OPTIMIZER_ORTOOLS_VERSION=${OPTIMIZER_ORTOOLS_VERSION}
 ARG REGISTRY=${REGISTRY:-registry.mapotempo.com/}
-ARG VROOM_VERSION=${VROOM_VERSION}
+ARG VROOM_VERSION=${VROOM_VERSION:-v1.8.0}
 
 # Install Vroom
-FROM ${REGISTRY}mapotempo/vroom:${VROOM_VERSION} as vroom
+FROM vroomvrp/vroom-docker:${VROOM_VERSION} as vroom
```

Comment:
This will have a conflict with https://github.com/Mapotempo/optimizer-api/pull/106 if it is merged before.

We need to chose which of these PR we want to merge first.